### DISCUSSION TITLE
android apk

### DISCUSSION THREAD

**Discussion by lordearl** [linagora/tmail-flutter#1110] [General]
What an exciting project!
Is the android apk available anywhere - ie on FDROID?

**Comment by hoangdat** [comment]
Hi, you can download it via Play store: https://play.google.com/store/apps/details?id=com.linagora.android.teammail.
Or download it via Github Actions the latest APK https://github.com/linagora/tmail-flutter/suites/8892645654/artifacts/406764465

**Reply by lordearl** [reply]
thanks - that's great
I am trying to test using a Fastmail JMAP account, do you happen to know what the JMAP URL would be for Fastmail?

**Reply by hoangdat** [reply]
ahh. At the moment, Fastmail JMAP not support to work with `username, password`. You can reference here: https://www.fastmail.com/developer/integrating-with-fastmail/. And, we have not had it in short term roadmap.

**Reply by lordearl** [reply]
Understood - do you know another provider who gives JMAP email accounts?

**Reply by hoangdat** [reply]
IMO, no. We have the plan to provide a SaaS offer in early of 2023. Hope it's useful for you

**Reply by lordearl** [reply]
Indeed! Count me in, anything I can do to help
